### PR TITLE
Quantization of Argmax

### PR DESCRIPTION
**Pull request by yihonglyu**

### CODE REVIEW

**Comment by yufenglee** [comment]
We also need to add a QDQ fusion support for case: Q->DQ->ArgMax to Q->ArgMax.

**Review by yufenglee — DISMISSED** [review]
:shipit:

File: onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc [hunk 1]
```diff
@@ -2181,6 +2181,46 @@ TEST(ReductionOpTest, ArgMax_int32_neg_axis) {
...
+                         5, 6,
+                         7, 8,
+
+                         9, 10,
+                         11, 12});
+  test.AddOutput<int64_t>("reduced", {3, 1, 2},
+                          {1, 1,
+                           1, 1,
+                           1, 1});
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});  // TensorRT: input/output with DataType Int8 in network without Q/DQ layers must have dynamic range set when no calibrator is used
```
Comment (edgchen1) [inline onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc:2201]: nit: line length

File: onnxruntime/test/python/quantization/test_op_argmax.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+        # make Conv node
+        conv_weight_name = 'conv_weight'
+        conv_weight_arr = np.random.randint(-1, 2, [32, 256, 1, 1]).astype(np.float32)
+        conv_weight_initializer = onnx.numpy_helper.from_array(conv_weight_arr, name=conv_weight_name)
+        conv_output_name = 'conv_output'
+        conv_inputs = [input_name, conv_weight_name]
+        conv_outputs = [conv_output_name]
+        conv_name = 'conv_node'
+        conv_node = onnx.helper.make_node('Conv', conv_inputs, conv_outputs, dilations=[1, 1], kernel_shape=[1, 1], pads=[0, 0, 0, 0], strides=[1, 1], name=conv_name)
```
Comment (edgchen1) [inline onnxruntime/test/python/quantization/test_op_argmax.py:48]: nit: line length

**Review by edgchen1 — APPROVED** [review]
kernel def hash changes look good to me

File: onnxruntime/test/python/quantization/test_op_argmax.py (hunk 2 above)
Comment (yihonglyu) [reply onnxruntime/test/python/quantization/test_op_argmax.py]: Fixed

File: onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc (hunk 1 above)
Comment (yihonglyu) [reply onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc]: Fixed

**Comment by yihonglyu** [comment]
@edgchen1 @yufenglee Please approve again with the line length fix

**Review by edgchen1 — APPROVED** [review]

**Pull request merged**